frontend/src/components/Checkbox/index.js

```diff
@@ -0,0 +1,29 @@
+import { PropTypes } from "prop-types";
+
+import styles from "./index.module.css";
+
+function Checkbox({ id, children }) {
```

Comment:
For the label text, I'd just use the prop `label` instead of `children`

The usecase is a bit different than for `button` and `h` headings